shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Is it possible that the UFS modules are loaded at runtime? At compile time, we don't need those modules, but only upon invocation when running the shell.